oval/redhat/types.go

```diff
@@ -0,0 +1,108 @@
+package redhat
```

Comment:
I'm curious is this a generated file? If so, how was it generated?

Reply:
No, it's manual. I tried some tools, but they don't work as I expected.
https://www.onlinetool.io/xmltogo/